server/src/test/java/org/elasticsearch/gateway/MetaDataStateFormatTests.java

```diff
@@ -212,26 +211,26 @@ public static void corruptFile(Path file, Logger logger) throws IOException {
                 bb.put(0, newValue);
                 raf.write(bb, filePointer);
                 logger.debug("Corrupting file {} --  flipping at position {} from {} to {} ", fileToCorrupt.getFileName().toString(),
-                    filePointer, Integer.toHexString(oldValue), Integer.toHexString(newValue));
```

Comment:
In this test we can inline `fileToCorrupt = file` (not something introduced here - I cannot comment on the line itself - but worth doing.)

Reply:
++

Reply:
https://github.com/elastic/elasticsearch/pull/35199/commits/b072c4f39f36dffc74a1f2f3c941a39f2857f1a3